openapi.yml

```diff
@@ -810,6 +884,32 @@ paths:
         $ref: "#/components/requestBodies/ProviderSetupStepCompleteRequest"
       tags:
         - Admin
+  /api/v1/admin/identity/sync:
+    parameters: []
+    post:
+      summary: Sync Identity
+      operationId: admin-post-api-v1-identity-sync
+      responses:
+        "200":
+          description: OK
+        "401":
+          $ref: "#/components/responses/ErrorResponse"
+        "500":
+          $ref: "#/components/responses/ErrorResponse"
+      description: Run the identity sync operation on demand
+  /api/v1/admin/identity:
+    get:
+      summary: Get identity configuration
+      tags: []
```

Comment:
nitpick - these should be tagged with the "Admin" tag so they get code-generated into the right frontend file